x-pack/plugins/dataset_quality/public/components/flyout/degraded_docs_trend/lens_attributes.ts

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DataView } from '@kbn/data-views-plugin/common';
+import type { TypedLensByValueInput } from '@kbn/lens-plugin/public';
+
+import { DEFAULT_LOGS_DATA_VIEW } from '../../../../common/constants';
+import {
+  flyoutDegradedDocsPercentageText,
+  flyoutDegradedDocsTrendText,
+} from '../../../../common/translations';
+
+const defaultDataView = {
+  id: `${DEFAULT_LOGS_DATA_VIEW}-id`,
+  title: DEFAULT_LOGS_DATA_VIEW,
+  timeFieldName: '@timestamp',
+} as DataView;
+
+export function getLensAttributes(color: string, dataView: DataView = defaultDataView) {
+  return {
+    visualizationType: 'lnsXY',
+    title: flyoutDegradedDocsTrendText,
+    references: [
+      {
+        id: dataView.id!,
+        name: 'indexpattern-datasource-current-indexpattern',
+        type: 'index-pattern',
+      },
+      {
+        id: dataView.id!,
```

Comment:
Why do we need a second layer with the same dataView?

Reply:
The second layer contains the doc count formula. The first and default layer is the time axis. Apparently, we need to define a reference for each configured layer.

Reply:
TIL, thanks mate!